src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1726,10 +1726,14 @@ public void LangVersion_LangVersions()
         public void LanguageVersionAdded_Canary()
         {
             // When a new version is added, this test will break. This list must be checked:
-            // - update the "UpgradeProject" codefixer
-            // - update all the tests that call this canary
-            // - update _MaxAvailableLangVersion (a relevant test should break when new version is introduced)
-            // - email release management to add to the release notes (see csharp-version in release.json in previous example: https://github.com/dotnet/core/pull/9493)
+            // 1. update the "UpgradeProject" codefixer
+            // 2. update all the tests that call this canary
+            // 3. update _MaxAvailableLangVersion (a relevant test should break when new version is introduced)
+            // 4. email release management to add to the release notes (see csharp-version in release.json in previous example: https://github.com/dotnet/core/pull/9493)
+            // 5. update the language history on csharplang
```

Comment:
Please link/update https://github.com/dotnet/csharplang/blob/main/Design-Process.md#steps-to-move-a-triaged-feature-to-an-implemented-feature as appropriate as well.